oonib/report/handlers.py

```diff
@@ -192,27 +140,63 @@ def parseNewReportRequest(request):
 
     return parsed_request
 
-class ReportHandler(OONIBHandler):
+def closeReport(report_id):
+    report_dir = FilePath(config.main.report_dir)
+    archive_dir = FilePath(config.main.archive_dir)
+    report_path = report_dir.child(report_id)
+    report_metadata_path = report_dir.child(report_id +
+                                            "-metadata.json")
+    print "Checking if %s exists and %s exists" % (report_path,
+                                                   report_metadata_path)
```

Comment:
Could you use the logger here? Python3 will arrive sooner or later and that day we'll be happier if there are less instances of the print statement to forward port to Python3.